CHANGELOG_PENDING.md

```diff
@@ -1,7 +1,7 @@
 ### Improvements
 
-- Add `--definition` flag to `esc env get` to output definition
-  [#416](https://github.com/pulumi/esc/pull/416)
```

Comment:
Did you mean to overwrite this previous entry in Improvements?

Reply:
yup this was from the previous release